app/controllers/api/design_files_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Api::DesignFilesController < ApplicationController
+  # PATCH/PUT /api/design_files/:designId
+  def update_last_modified
+    design_file = DesignFile.find(params[:designId])
+    design_file.update!(last_modified: Time.current)
```

Comment:
Consider adding input validation for `params[:designId]` to ensure it is a valid ID. This will help prevent potential security issues such as SQL injection.